src/rt/lifetime.d

```diff
@@ -12,17 +12,12 @@
 
 module rt.lifetime;
 
-import core.stdc.stdlib;
-import core.stdc.string;
-import core.stdc.stdarg;
-import core.bitop;
-import core.memory;
 debug(PRINTF) import core.stdc.stdio;
 static import rt.tlsgc;
 
+import core.memory;
```

Comment:
Can this one stay at line 19?

Reply:
Sure, I can move it. What is the reason for this?

Reply:
Oh, sorry, I meant line 19 in the previous version (so, leave it behind `debug(PRINTF) import core.stdc.stdio`). No particular reason except minimizing the diff (because there's no reason as far as I can see for this move), and if you move it back it keeps the imports sorted alphabetically.